dkg/pedersen/handler.go

```diff
@@ -285,6 +314,11 @@ func (h *Handler) start(start Start, receivedDeals []Deal, from mino.Address,
 	h.privShare = distrKey.PriShare()
 	h.Unlock()
 
+	h.startRes.Lock()
+	h.startRes.distrKey = distrKey.Public()
+	h.startRes.participants = start.addresses
+	h.startRes.Unlock()
```

Comment:
Can you move that to a function to be consistent ? In case the state is moved to a different package for instance.